utilities/CMakeLists.txt

```diff
@@ -29,7 +29,7 @@ find_package(FFmpeg QUIET)
 if(OpenCL_FOUND)
     add_subdirectory(runvx)
     add_subdirectory(runcl)
-    add_subdirectory(loom_shell)
```

Comment:
Why was this module turned off? @rrawther @paveltc @shobana-mcw

Reply:
We commented it because we were facing build issues and its machine-specific. By mistake it was added in PR. It can be uncommented.